specification/cognitiveservices/data-plane/Language/preview/2023-11-15-preview/analyzedocuments.json

```diff
@@ -0,0 +1,2237 @@
+            "description": "Age"
+          },
+          {
+            "name": "PhoneNumber",
+            "value": "PhoneNumber",
+            "description": "Phone Number"
+          },
+          {
+            "name": "IPAddress",
+            "value": "IPAddress",
+            "description": "IP Address"
+          },
+          {
+            "name": "Date",
+            "value": "Date",
+            "description": "Date"
+          },
+          {
+            "name": "Person",
+            "value": "Person",
+            "description": "Person"
+          },
+          {
+            "name": "Address",
+            "value": "Address",
+            "description": "Address"
+          },
+          {
+            "name": "All",
+            "value": "All",
+            "description": "All PII categories."
+          },
+          {
+            "name": "Default",
+            "value": "Default",
+            "description": "Default PII categories for the language."
+          }
+        ]
+      }
+    },
+    "PiiTaskParameters": {
+      "type": "object",
+      "description": "Supported parameters for a PII Entities Recognition task.",
+      "properties": {
+        "domain": {
+          "$ref": "#/definitions/PiiDomain"
+        },
+        "piiCategories": {
+          "$ref": "#/definitions/PiiCategory"
+        },
+        "excludePiiCategories": {
+          "$ref": "#/definitions/PiiCategoriesExclude"
+        },
+        "stringIndexType": {
+          "$ref": "common.json#/definitions/StringIndexType"
+        },
+        "redactionCharacter": {
+          "$ref": "#/definitions/RedactionCharacter"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "common.json#/definitions/PreBuiltTaskParameters"
+        }
+      ]
+    },
+    "PiiDomain": {
+      "type": "string",
+      "description": "The PII domain used for PII Entity Recognition.",
+      "default": "none",
+      "enum": [
+        "phi",
+        "none"
+      ],
+      "x-ms-enum": {
+        "name": "PiiDomain",
+        "modelAsString": true,
+        "values": [
+          {
+            "name": "phi",
+            "description": "Indicates that entities in the Personal Health Information domain should be redacted.",
+            "value": "phi"
+          },
+          {
+            "name": "none",
+            "description": "Indicates that no domain is specified.",
+            "value": "none"
+          }
+        ]
+      }
+    },
+    "piiCategories": {
+      "type": "array",
+      "description": "Enumeration of PII categories to be returned in the response.",
+      "items": {
+        "$ref": "#/definitions/PiiCategory"
+      }
+    },
+    "PiiCategoriesExclude": {
+      "description": "(Optional) describes the PII categories to return",
+      "items": {
+        "type": "string",
+        "x-ms-enum": {
+          "name": "PiiCategoriesExclude",
+          "modelAsString": true
+        },
+        "enum": [
```

Comment:
Why does `piiCategories` use the `PiiCategory` enum while `PiiCategoriesExclude` does not? Is it a subset?

Reply:
PIIcategoriesExclude is a subset of PIIcategories. Its more cleanly expressed via the tsp but the swagger generated will be similar to what we have here.